### PR TITLE
add standard set of icons

### LINKED ISSUE #410: Add new Favicon
New asset attached
wc_favicon

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What is this PR trying to achieve?
Fixes #410 (well at least does what it says, there is still issue around the favicon not being clean in context of homescreens etc)

## What does it look like?
screen shot 2017-02-07 at 15 58 54

## For interface components
- [x] Browser testing - Have you checked the component in our supported browsers
- [x] No javascript - Have you checked the component with javascript disabled
- [x] A11y testing - Have you run pa11y against the component and fixed/verified all errors, warnings and notices?

### CODE REVIEW

File: server/views/layout/default.njk [hunk 1]
```diff
@@ -2,13 +2,20 @@
...
     <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
     <title>{{ pageConfig.title + ' | ' if pageConfig.title }}Wellcome Collection</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     {% include "partials/css-head.njk" %}
-    <link rel="stylesheet" href="/assets/css/{{ 'application.css' | getCacheBustFile }}" />
+    <link rel="stylesheet" href="/assets/css/{{ 'application.css' | getCacheBustFile }}"/>
     {% component 'head-js', { file:  'app.js' | getCacheBustFile  } %}
     {# TODO: Add apple icons, different sizes icons etc #}
```
Comment (gestchild) [inline server/views/layout/default.njk]: Probably don't need this now

Comment (gestchild) [inline server/views/layout/default.njk]: Probably don't need this comment now

**Comment by jamesgorrie** [comment]
> particular sizes

I use realfavicongenerator.net - they keep it up to date. [There's a good article on it here](https://css-tricks.com/favicon-quiz/).

> AppCache

That's been deprecated in favour of service workers - which is a definite 👍

**Comment by gestchild** [comment]
Good point, meant service workers (not that I've used either before)

**Pull request merged**